### ISSUE TITLE
SWAP grows until OOM in Ubuntu 22.04

### ISSUE THREAD

**Issue by gAlleb** [astrada/google-drive-ocamlfuse#826]
Hi! Thank you for the app!

So I was was using the latest version over ppa on Ubuntu 20.04 with default settings, headless (as a source of music for 5 24/7 radios - icecast, hls, etc.). And everything has been fine! 

Then I upgraded to Ubuntu 22.04 and upgraded ppa and package also to `deb http://ppa.launchpad.net/alessandro-strada/ppa/ubuntu jammy main` and my swap started growing progressively until OOM comes ;) Usually takes a day, day and a half. Tried a lot of different tricks — nothing worked.

So I switched ppa back to `deb http://ppa.launchpad.net/alessandro-strada/ppa/ubuntu focal main` and reinstalled focal version on current OS (Ubuntu 22.04) and everything went back to normal: RAM usage is low, swap even lower.

Maybe it is a bug or something, I don't know. Just decided to share if someone would have similar issue.

**Comment by frederik-elwert** [comment]
Yes, I can confirm this. Downloading large files was using a lot of RAM and generally very slow, switching to the `focal` version fixed this.
